DownloadManager/Plugin.DownloadManager.UWP/DownloadFileImplementation.cs

```diff
@@ -131,6 +144,7 @@ public DownloadFileImplementation(DownloadOperation downloadOperation)
                 file = await folder.CreateFileAsync(downloadUrl.Segments.Last(), CreationCollisionOption.GenerateUniqueName);
             }
 
+            DestinationPathName = file.Path;
```

Comment:
This would now be the only instance where this value is set before the download is started ... can we get this consistent? All other platforms should also report the destination when the download is complete (but before the status is updated).

Another question is: Could you please check and confirm how this behaves if the app crashes? Does the download still continue or does it (unlike in the implementation on iOS or Android) stop?

If it continues, is this path still available if the download finishes then?

Reply:
I tried, but the ProgressChanged method actually kicked in after the Status property changed... Though I just noticed that way my mistake, lol. Fixed.

Reply:
Do you have a way for me to crash the app whilst running?

Reply:
This should create a null-reference exception. Put it at the right place and it will get the app to crash 😉

```
object o = null;
DateTime d = (DateTime)o;
```

Reply:
I know, but that instantly kills the process since Visual Studio doesn't want to continue. I thought you had a way to crash and keep the process running. Or was that the question? Well then no, the app crashes and the download does not complete. At least in Debug mode.

I published the app with the Release configuration and side-loaded it to see whether the download would continue or stop. I added to the code that it would create a text file containing the download location, but it never shows up. So I guess it's safe to assume that the answer is no.